Xamarin.Forms.Platform.iOS/Material/MaterialButtonRenderer.cs

```diff
@@ -94,9 +86,20 @@ protected override void OnElementChanged(ElementChangedEventArgs<Button> e)
 			}
 		}
 
+		protected virtual ButtonScheme CreateButtonScheme()
+		{
+			return new ButtonScheme
+			{
+				TypographyScheme = new TypographyScheme(),
+				ColorScheme = MaterialColors.Light.CreateColorScheme()
+			};
+		}
```

Comment:
The iOS Material design is set via the Themer applying a Scheme. In this case, we are using the "light" theme with values copied from Android.